pgvectorscale/src/util/mod.rs

```diff
@@ -121,6 +121,13 @@ impl ItemPointer {
             len: len as _,
         }
     }
+
+    pub fn ip_distance(self, other: Self) -> usize {
```

Comment:
This may be common knowledge for folks in this domain, but it may be valuable to add some docs here on what this does, expected behavior, the fact that there is an assert and thus potential panicking behavior.